lib/diff_web/controllers/page_controller.ex

```diff
@@ -36,14 +36,29 @@ defmodule DiffWeb.PageController do
     end
   end
 
-  defp parse_versions(versions) do
-    with [from, to] <- String.split(versions, "..", trim: true),
-         {:ok, from} <- Version.parse(from),
-         {:ok, to} <- Version.parse(to) do
+  defp parse_versions(package, versions) do
+    with [from, to] <- versions |> String.split("..") |> Enum.map(&String.trim/1),
+         {:ok, from} <- parse_version(package, :from, from),
+         {:ok, to} <- parse_version(package, :to, to) do
       {:ok, to_string(from), to_string(to)}
     else
       _ ->
         :error
     end
   end
+
+  defp parse_version(_package, :from, ""), do: :error
+
+  defp parse_version(package, :to, "") do
+    with {:ok, versions} <- Diff.Package.Store.get_versions(package) do
+      version =
+        versions
+        |> Enum.map(&Version.parse!/1)
+        |> Enum.max()
```

Comment:
We need to use the new `module.compare/2` sorting instead of term based sorting.

```suggestion
        |> Enum.max(Version)
```

Reply:
Nice, I've changed in here https://github.com/hexpm/diff/pull/45/commits/b9071e34c235a14886d5f9aa03c0bf3ab8a63e0f 👍